cves/CVE-2017-14058.yml

```diff
@@ -129,7 +128,11 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: The lifetime of this was pretty long. This went from 2011 to 2017 unnoticed. This had many different developers and new features. As to fix this issue 
```

Comment:
Spacing might be an issue here. They recommend we keep YAML text lines to a maximum of 80 characters.

Reply:
thanks fixed the spacing